### PR TITLE
Node ssh command

### PR DESCRIPTION
**Pull request by felipemadero**
https://asciinema.org/a/619916

### CODE REVIEW

File: cmd/nodecmd/list.go [hunk 1]
```diff
@@ -40,34 +39,32 @@ func list(_ *cobra.Command, _ []string) error {
 	if len(clustersConfig.Clusters) == 0 {
 		ux.Logger.PrintToUser("There are no clusters defined.")
 	}
-	for clusterName, clusterConfig := range clustersConfig.Clusters {
-		ux.Logger.PrintToUser(fmt.Sprintf("Cluster %q", clusterName))
+	for clusterName, clusterConf := range clustersConfig.Clusters {
+		ux.Logger.PrintToUser("Cluster %q (%s)", clusterName, clusterConf.Network.Kind.String())
```
Comment (arturrez) [inline cmd/nodecmd/list.go:43]: use Name()

File: cmd/nodecmd/list.go [hunk 2]
```diff
@@ -40,34 +39,32 @@ func list(_ *cobra.Command, _ []string) error {
...
 			return err
 		}
 		if err := setupAnsible(clusterName); err != nil {
 			return err
 		}
+		ansibleHostIDs, err := ansible.GetAnsibleHostsFromInventory(app.GetAnsibleInventoryDirPath(clusterName))
+		if err != nil {
+			return err
+		}
 		ansibleHosts, err := ansible.GetHostMapfromAnsibleInventory(app.GetAnsibleInventoryDirPath(clusterName))
```
Comment (arturrez) [inline cmd/nodecmd/list.go:54]: is ansibleHosts used? should be removed

File: cmd/nodecmd/ssh.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/utils"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/spf13/cobra"
+)
+
+func newSSHCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "ssh [clusterName] [cmd]",
+		Short: "(ALPHA Warning) Execute ssh command on node/s",
```
Comment (arturrez) [inline cmd/nodecmd/ssh.go:19]: replace /s with (s)

File: cmd/nodecmd/ssh.go [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+		var err error
+		clustersConfig := models.ClustersConfig{}
+		if app.ClustersConfigExists() {
+			clustersConfig, err = app.LoadClustersConfig()
+			if err != nil {
+				return err
+			}
+		}
+		if len(clustersConfig.Clusters) == 0 {
+			ux.Logger.PrintToUser("There are no clusters defined.")
```
Comment (arturrez) [inline cmd/nodecmd/ssh.go:44]: we should return there
no point of continue

File: cmd/nodecmd/ssh.go [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+		}
+		for clusterName, clusterConfig := range clustersConfig.Clusters {
+			ux.Logger.PrintToUser("Cluster %q (%s)", clusterName, clusterConfig.Network.Kind.String())
```
Comment (arturrez) [inline cmd/nodecmd/ssh.go:47]: use Name()

File: cmd/nodecmd/ssh.go [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+	}
+	ansibleHostIDs, err := ansible.GetAnsibleHostsFromInventory(app.GetAnsibleInventoryDirPath(clusterName))
+	if err != nil {
+		return err
+	}
+	ansibleHosts, err := ansible.GetHostMapfromAnsibleInventory(app.GetAnsibleInventoryDirPath(clusterName))
+	if err != nil {
+		return err
+	}
+	for _, host := range ansibleHostIDs {
```
Comment (arturrez) [inline cmd/nodecmd/ssh.go:74]: you execute this commands once after another. 
Do you want to use waitGroup or errorGroup to execute ssh commands in parallel ? 
maybe add flags to make these commands executed in parallel so user has a choice

File: cmd/nodecmd/ssh.go [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+		_, cloudID, err := models.HostAnsibleIDToCloudID(host)
+		if err != nil {
+			return err
+		}
+		cmdLine := utils.GetSSHConnectionString(ansibleHosts[host].IP, ansibleHosts[host].SSHPrivateKeyPath) + " " + strings.Join(args[1:], " ")
```
Comment (arturrez) [inline cmd/nodecmd/ssh.go:79]: use fmt.Sprintf instead

File: cmd/nodecmd/ssh.go [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+		ux.Logger.PrintToUser("%s[%s] %s", indent, cloudID, cmdLine)
+		if len(args) > 1 {
+			splitCmdLine := strings.Split(cmdLine, " ")
+			cmd := exec.Command(splitCmdLine[0], splitCmdLine[1:]...) //nolint: gosec
```
Comment (arturrez) [inline cmd/nodecmd/ssh.go:83]: we should set cmd environment with os.environ to be consistent with what user shell env is when he executed ssh commands from his terminal manually

File: cmd/nodecmd/list.go (hunk 2 above)
Comment (felipemadero) [reply cmd/nodecmd/list.go:54]: it is used to get IPs

Comment (felipemadero) [reply cmd/nodecmd/list.go:54]: node list is showing IPs anyway

**Review by arturrez — APPROVED** [review]

**Pull request merged**